e2e/packages/sync-test/package.json

```
@@ -34,6 +34,6 @@
     "viem": "2.9.20",
     "vite": "^4.2.1",
     "vitest": "0.34.6",
-    "zod": "^3.22.2"
+    "zod": "^3.23.8"
```

since [other issues](https://github.com/colinhacks/zod/issues/2697#issuecomment-2159467187) point to problems with mismatching zod versions, I wonder if we should pin this to a specific version number rather than a range
```suggestion
    "zod": "3.23.8"
```

---

and could also move this to `catalog:` like we have for ark deps
https://github.com/latticexyz/mud/commit/791736c86cc2cc4263dc4c3da29b44c8546a1e66#diff-18ae0a0fab29a7db7aded913fd05f30a2c8f6c104fadae86c9d217091709794cR5-R6

---

agreed, pinning makes sense, pinned all packages.